### ISSUE TITLE
TR support

### ISSUE THREAD

**Issue by thesuperfatcat** [YaoFANGUK/video-subtitle-extractor#80]
Hello, it is possible to have turkish support?

Thx.

**Comment by YaoFANGUK** [comment]
do you mean Turkish interface or Turkish subtitle extraction

**Comment by thesuperfatcat** [comment]
Yes, turkish subtitle extraction. English mode can subtitle extraction for turkish but lacks character support

**Comment by YaoFANGUK** [comment]
Turkish corpus is needed in order to support turkish subtitle extraction. Collecting corpus is a very time-consuming work. I'm sorry I might have no time to do this. This repo needs contribution from more people.
